thrall/app/lib/kinesis/ThrallEventConsumer.scala

```diff
@@ -45,7 +45,8 @@ class ThrallEventConsumer(es: ElasticSearchVersion,
 
       val updateMessage = Json.parse(message).as[UpdateMessage] // TODO validation
       val timestamp = r.getApproximateArrivalTimestamp
-      Logger.info("Got update message (" + timestamp + "): " + updateMessage.subject + "/" + updateMessage.id)
+      val idForLogging = Seq(updateMessage.id, updateMessage.image.map(_.id)).flatten
+      Logger.info("Got update message (" + timestamp + "): " + updateMessage.subject + "/" + idForLogging.mkString(" "))
```

Comment:
Would it make sense to include this part in line 48? `idForLogging.mkString(" ")`